### ISSUE TITLE
ENH/API/DOC: checkout performance options for large data sets (>500 trials at high sampling frequency)

### ISSUE THREAD

**Issue by dengemann** [autoreject/autoreject#22]
So far we had discussed

1) subsampling of fitting and interpolating epochs for data augmentation
2) fit on k epochs, transform all

The task here is to at least show how to use autoreject when data are massive.

cc @fraimondo @jasmainak @agramfort

**Comment by jasmainak** [comment]
Thanks @dengemann . I'd say we don't particularly need a new dataset as we can already see these issues on existing examples. 

@fraimondo, if you want to give a shot at making [this example](http://autoreject.github.io/auto_examples/plot_visualize_bad_epochs.html#sphx-glr-auto-examples-plot-visualize-bad-epochs-py) run faster without making too much of an abrupt change in the codebase, that would be really welcome

**Comment by dengemann** [comment]
>  I'd say we don't particularly need a new dataset as we can already see these issues on existing examples.

yes, and if needed we can just tile existing data, making super long epochs of epochs.

> , if you want to give a shot at making this example run faster

+1
